bundles/pixi.js/src/index.ts

```diff
@@ -2,7 +2,7 @@ import '@pixi/polyfill';
 
 import * as utils from '@pixi/utils';
 import { AccessibilityManager } from '@pixi/accessibility';
-import { InteractionManager } from '@pixi/interaction';
+// import { InteractionManager } from '@pixi/interaction';
```

Comment:
Would like to leave **pixi.js** as is for v6.

Reply:
Ah, yes. This was just to demonstrate which examples still work as-is with the EventSystem. I'll revert this.

Reply:
Reverted 👍